### PR TITLE
landingpage

### PR DESCRIPTION
**Pull request by preethasri**
created landing page for notes app
Screenshot (20)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,6 +24,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link
+    href="https://fonts.googleapis.com/css?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Round"
+    rel="stylesheet"
+/>
+
     <title>React App</title>
   </head>
```
Comment (Sharmaryan) [inline public/index.html:33]: fix title

File: src/components/navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    flex:1;
+    display:flex;
+    align-items: center;
+    justify-content: flex-start;
+    margin:0 -15px;
+    
+}
+.navbar-left .nav-item i{
+    margin-left: 20px;
+}
```
Comment (Sharmaryan) [inline src/components/navbar/navbar.css:39]: you can use rems here, as you are already using it in your code

File: src/pages/landingpage/landingpage.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    padding:10px 10px;
+   max-width: fit-content;
+   font-size: 1.2rem;
+   margin-top:2rem;
+   display: flex;
+   align-items: center;
+   justify-content: center;
+   background-color: var(--primary-color);
+   color:white;
+   cursor:pointer;
```
Comment (Sharmaryan) [inline src/pages/landingpage/landingpage.css:41]: you have used color white Two times till now I think, so you can use variables for that

**Review by Sharmaryan — APPROVED** [review]
UI looks clean , I left few suggestions

**Review by Varu98 — APPROVED** [review]

File: src/pages/landingpage/landingpage.css (hunk 3 above)
Comment (preethasri) [reply src/pages/landingpage/landingpage.css:41]: will work on this aryan

**Pull request merged**